content/pages/disability-benefits/apply-for-benefits/application.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Apply for disability benefits
```

Comment:
I was waiting to hear back to see the design specs, so I'm not entirely sure if it should be both, but I'll change it to tracking benefits for now.